README.md

```diff
@@ -56,7 +56,7 @@ $ docker run -p 8000:8000 -p 8001:8001 ghcr.io/arroyosystems/arroyo:single
 ```
 
 This will run all of the core arroyo services and a temporary Postgres database. The web UI will be exposed at
-http://localhost:8000.
+http://localhost:8000. If you're on an arm64 machine, instead use ghcr.io/arroyosystems/arroyo:single-arm64.
```

Comment:
Can you move this before the previous line, so that it's right after the bash statement?

This is great for now, but we'll probably want to do a proper multi-arch image (https://www.docker.com/blog/multi-arch-build-and-images-the-simple-way/) so that users don't need to pick the right one.